src/Shared/Recruit.Shared.Web/RuleTemplates/RuleTemplateRunner.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Esfa.Recruit.Shared.Web.RuleTemplates
+{
+    public class RuleTemplateRunner : IRuleTemplateRunner
+    {
+        private readonly IServiceProvider _provider;
+
+        public RuleTemplateRunner(IServiceProvider provider)
+        {
+            _provider = provider;
+        }
+
+        public string ToText(object data, string fieldName)
+        {
+            if (data == null)
```

Comment:
Where data is null, may be there is a static message.

Reply:
I think returning an empty string is OK here.  I'll filter out results with an empty message instead when building the VM (This should never happen btw).